src/components/snapshots/takeSnapshot.ts

```diff
@@ -55,7 +55,7 @@ const takeSnapshot: SnapshotOperation = async ({
     // Write snapshot and config to folder
     await fs.writeFile(
       path.join(newSnapshotFolder, `${SNAPSHOT_FILE_NAME}.json`),
-      JSON.stringify(snapshotObject, null, ' ')
+      JSON.stringify(snapshotObject, null, 2)
```

Comment:
This is my attempt to make the json format saved by the snapshot exporter the same as our editor auto-formatter. It's not exactly the same, but it's closer, I think.